kotest-assertions/kotest-assertions-json/src/jvmMain/kotlin/io/kotest/assertions/json/JsonConverters.kt

```diff
@@ -0,0 +1,40 @@
+package io.kotest.assertions.json
+
+import io.kotest.assertions.failure
+import kotlin.contracts.ExperimentalContracts
+import kotlin.contracts.contract
+import kotlin.reflect.typeOf
+
+/**
+ * Returns encoded JSON.
+ *
+ * We need to distinct `null` and `"null"`. The first one means there is no JSON at all, the second is null in JSON.
+ * So we can't do just `Json.toString()`.
+ */
+val Json?.representation
```

Comment:
Does this need to stand public? We can unclutter a little bit the `Json` namespace by making this private/internal

Reply:
I've made it internal in 2c2eebb using your suggestion about PublishedApi.